x-pack/plugins/security_solution/public/timelines/components/timeline/body/data_driven_columns/stateful_cell.test.tsx

```diff
@@ -91,7 +88,7 @@ describe('StatefulCell', () => {
         isDetails: false,
         linkValues,
         rowIndex: ariaRowindex - 1,
-        timelineId: `${timelineId}-${tabType}`,
+        timelineId: `${timelineId}`,
```

Comment:
```suggestion
        timelineId,
```

No need for the string interpolation anymore